apps/aecore/src/aec_db.erl

```diff
@@ -581,7 +546,7 @@ add_backend_plugins(_) ->
     ok.
 
 add_index_plugins() ->
-    mnesia_schema:add_index_plugin({acct2tx}, aec_db, ix_acct2tx).
```

Comment:
Shall otp_patches and any reference to it (e.g. rebar.config, `-pa patches/ebin` in vm.args files) be contextually removed?

Reply:
We can leave it for a while before we determine that we don't need it. There is infrastructure in state channels to use them (although it is not currently used).